bladex/blade.py

```diff
@@ -313,16 +313,17 @@ def apply_transformations(self, reflect=True):
 
         self._planar_to_cylindrical()
 
-    def rotate(self, deg_angle=None, rad_angle=None):
+    def rotate(self, deg_angle=None, rad_angle=None, axis='x'):
         """
-        3D counter clockwise rotation about the X-axis of the Cartesian
+        3D counter clockwise rotation about the specified axis of the Cartesian
         coordinate system, which is the axis of rotation of the propeller hub.
 
         The rotation matrix, :math:`R(\\theta)`, is used to perform rotation
-        in the 3D Euclidean space about the X-axis, which is -- by default --
-        the propeller axis of rotation.
+        in the 3D Euclidean space about the specified axis, which is
+        -- by default -- the x axis.
 
-        :math:`R(\\theta)` is defined by:
+        :math: when the axis of rotation is the x-axis `R(\\theta)` is defined
```

Comment:
I think you have to keep the :math: keyword attached to the mathematical expression